package.json

```diff
@@ -13,20 +13,29 @@
     "prestorybook": "rnstl"
   },
   "dependencies": {
+    "@react-native-async-storage/async-storage": "^1.15.4",
+    "@codler/react-native-keyboard-aware-scroll-view": "^1.0.1",
+    "@react-native-community/async-storage": "^1.12.1",
+    "@react-native-community/picker": "^1.8.1",
```

Comment:
Remove package if unused

Reply:
I didn't delete the "@codler/react-native-keyboard-aware-scroll-view": "^1.0.1" because I am using it with quiting keyboard.